app/repositories/queue_repository.rb

```
@@ -33,6 +33,45 @@ def self.appeals_from_tasks(tasks)
     appeals
   end
 
+  # :nocov:
+  def self.find_case(vacols_id, css_id)
+    VACOLS::CaseDecision.find_by_vacols_id_and_css_id(vacols_id, css_id)
+  end
+  # :nocov:
+
+  # :nocov:
+  def self.reassign_case_to_judge(decision_record:, judge_css_id:, decision_hash:)
+    return if !decision_record || !judge_css_id
```

Let me know if we actually want to raise an error here if decision_record is nil or if judge_css_id is nil